nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -25,6 +26,22 @@ def check_processing_module_name(processing_module: ProcessingModule):
         )
 
 
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=NWBFile)
+def check_session_start_time(nwbfile: NWBFile):
+    """Check if the session_start_time was set to an appropriate value."""
+    if nwbfile.session_start_time <= datetime(1980, 1, 1).astimezone():
+        return InspectorMessage(message="The session_start_time may not be set to the true date of the recording.")
+
+
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=NWBFile)
+def check_timestamps_reference_time(nwbfile: NWBFile):
+    """Check if the timestamps_reference_time was set to an appropriate value."""
+    if nwbfile.timestamps_reference_time <= datetime(1980, 1, 1).astimezone():
+        return InspectorMessage(
+            message="The timestamps_reference_time may not be set to the true date of the recording."
+        )
```

Comment:
If this field is used, it's fairly common to use Jan 1 1970. Is that a problem?

Reply:
Hmmm I suppose for `timestamps_reference_time` we can let it slide since, as long as all the time values in the value are indeed aligned to whatever it happens to be, it could be anything they want.